lib/rmt/downloader.rb

```diff
@@ -85,24 +99,35 @@ def make_request(remote_file, local_file, request_fiber, checksum_type = nil, ch
 
     downloaded_file = Tempfile.new('rmt', Dir.tmpdir, mode: File::BINARY, encoding: 'ascii-8bit')
 
+    headers = {}
+    headers['If-Modified-Since'] = if_modified_since if if_modified_since
```

Comment:
if if if if? Would read nicer to me if the variable name wouldn't start with "if_" ... still voting for renaming it to something more nice :name_badge:

Reply:
:rofl: Yes, that turned out a bit weird.